tests/integration/cassandra_tests.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import time
+import unittest
+
+try:
+    from cassandra import InvalidRequest
+    from cassandra.cluster import Cluster
+    from cassandra.query import dict_factory, named_tuple_factory
+except ImportError:
+    raise unittest.SkipTest("cassandra-driver is not installed")
```

Comment:
Might not be any way around it, but are you concerned that people could unintentionally miss errors because of this? Could `cassandra-driver` be a required dependency?